server/database/queries/house/updateHouse.js

```diff
@@ -0,0 +1,35 @@
+const connection = require('../../config/connection');
+
+const updateHouse = ({
+  roomNo,
+  category,
+  price,
+  description,
+  title,
+  locationId,
+  bathNo,
+  area,
+  image,
+  userId,
+  houseId,
+}) => {
+  const sql = {
+    text:
+      'UPDATE houses SET room_num=$1 ,category=$2 , price=$3 ,description = $4,title=$5 ,location_id = $6 ,bathroom_num=$7 ,area = $8, image=$9 WHERE user_id =$10 AND id=$11;',
+    values: [
```

Comment:
I think you need to double check these - the values don't match the placeholders. It looks OK until bathroom_num but looks like the ones after that were rearranged